Helper_Scripts/validate_hosted_saas_profile.py

```
@@ -0,0 +1,163 @@
+- PostgreSQL-backed AuthNZ storage
+- production guards enabled
+- a public frontend origin for auth emails
+- billing redirect hardening locked to the public app host
+"""
+
+from __future__ import annotations
+
+import os
+import sys
+from dataclasses import dataclass, field
+from typing import Mapping
```

Ruff (0.15.6)

[warning] Import from `collections.abc` instead: `Mapping`

Import from `collections.abc`

(UP035)